components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -206,8 +210,15 @@ ic_tod="0" sim_year="2000" glc_nec="0" use_crop=".true." irrigate=".true." nu_co
 <finidat hgrid="ne0np4_enax4v1" sim_year="2000">lnd/clm2/initdata_map/clmi.ICRUCLM45SP.2000-01-01.enax4v1_oRRS18to6_simyr2000_c180430.nc</finidat>
 <finidat hgrid="ne0np4_twpx4v1" ic_ymd="101" sim_year="2000">lnd/clm2/initdata_map/clmi.ICRUCLM45SP.2000-01-01.twpx4v1_oRRS18to6v3_simyr2000_c180430.nc</finidat>
 
-<!-- for present day simulations - year 2000 -->
+<!-- for present day simulations - year 2010 -->
+<fsurdat hgrid="ne30np4"   sim_year="2010" use_crop=".false." >
+lnd/clm2/surfdata_map/surfdata_ne30np4_simyr2010_c20181025.nc</fsurdat>
```

Comment:
Just a note for record, though not the purpose here, with this PR, surfdata file for ne30np4 will change from surfdata_ne30np4_simyr2000_c190730.nc to  surfdata_ne30np4_simyr2010_c20181025.nc